components/widget/match/ticker/widget_match_ticker_container.lua

```diff
@@ -0,0 +1,113 @@
+---
+-- @Liquipedia
+-- wiki=commons
+-- page=Module:Widget/Match/Ticker/Container
+--
+-- Please see https://github.com/Liquipedia/Lua-Modules to contribute
+--
+
+local Array = require('Module:Array')
+local Class = require('Module:Class')
+local Lua = require('Module:Lua')
+local Operator = require('Module:Operator')
+local Template = require('Module:Template')
+
+local Widget = Lua.import('Module:Widget')
+local HtmlWidgets = Lua.import('Module:Widget/Html/All')
+local FilterConfig = Lua.import('Module:FilterButtons/Config')
+
+---@class MatchTickerContainer: Widget
+---@operator call(table): MatchTickerContainer
+local MatchTickerContainer = Class.new(Widget)
+MatchTickerContainer.defaultProps = {
+	upcomingTemplate = 'MainPageMatches/Upcoming',
+	completedTemplate = 'MainPageMatches/Completed',
```

Comment:
[oos] i think eventually we should switch that to module calls

Reply:
Yes and no, it's will be needed  regardless for template expand via the API call that happens when toggling filter buttons